decidim-elections/app/commands/decidim/elections/admin/setup_election.rb

```diff
@@ -73,8 +73,8 @@ def election_data
             description: {
               name: {
                 text: [{
-                  value: election.title[I18n.locale.to_s],
-                  language: I18n.locale.to_s
+                  value: election.title[I18n.default_locale.to_s],
```

Comment:
Should this be using the `organization.default_locale` value? I'm thinking of multitenant installations...

Reply:
Sure, I'll change it. Thank you!